modules/cloud-guides/pages/ingest-catalog-data-teradata-s3-with-glue.adoc

```diff
@@ -0,0 +1,282 @@
+
+
+# Job function abstraction
+def process_table(table_name, transformation_ctx_prefix, catalog_database, catalog_table_name):
+    dynamic_frame = glueContext.create_dynamic_frame.from_options(
+        connection_type="teradata",
+        connection_options={
+            "dbtable": table_name,
+            "connectionName": "Teradata connection default",
+            "query": f"SELECT TOP 1 * FROM {table_name}", # This line can be modified to ingest the full table or rows that fulfill an specific condition
+        },
+        transformation_ctx=transformation_ctx_prefix + "_read",
+    )
+
+    s3_sink = glueContext.getSink(
+        path=target_s3_bucket,
+        connection_type="s3",
+        updateBehavior="UPDATE_IN_DATABASE",
+        partitionKeys=[],
+        compression="snappy",
+        enableUpdateCatalog=True,
+        transformation_ctx=transformation_ctx_prefix + "_s3",
+    )
+    # Dynamically set catalog table name based on function parameter
+    s3_sink.setCatalogInfo(
+        catalogDatabase=catalog_database, catalogTableName=catalog_table_name
+    )
+    s3_sink.setFormat("csv")
+    s3_sink.writeFrame(dynamic_frame)
+
+
+# Job execution section
+for table_name in table_names:
+    full_table_name = f"{database_name}.{table_name}"  
+    transformation_ctx_prefix = f"{database_name}_{table_name}"  
+    catalog_table_name = f"{table_name}_catalog"  
+    # Call your process_table function for each table
+    process_table(full_table_name, transformation_ctx_prefix, catalog_database_name, catalog_table_name)
+
+job.commit()
+----
+
+* Assign a name to your script
++
+image::{dir}/Glue-script-3.PNG[script in editor,align="center",width=80%]
+
+* In Job details, Basic properties:
+** Select the IAM role you created for the ETL job.
+** For testing, select "2" as the Requested number of workers, this is the minimum allowed.
++
+image::{dir}/Glue-script-4.PNG[script configurations,align="center",width=80%]
+** In `Advanced properties`, `Connections` select your connection to Teradata Vantage. 
++
+TIP: The connection created must be referenced twice, once in the job configuration, once in the script itself.
++
+image::{dir}/Glue-script-5.PNG[script configuration connection,align="center",width=80%]
+* Click on `Save`.
+* Click on `Run`.
+** The ETL job takes a couple of minutes to complete, most of this time is related to starting the Spark cluster.
+
+== Checking the Results
+
+* After the job is finished:
+** Go to Data Catalog, Databases.
+** Click on the catalog database you created.
+** In this location, you will see the tables extracted and cataloged through your Glue ETL job.
++
+image::{dir}/Results.PNG[result tables,align="center",width=80%]
+
+* All tables ingested are also present as compressed files in S3. Rarely, these files would be queried directly. Services such as AWS Athena can be used to query the files relying on the catalog metadata.
+
+== Summary
```

Comment:
You can also add links pointing to contextual internal resources like other quick-starts or docs.

Reply:
I thought on this but usually we add those in the Further Reading.
 
For this subject we didn't have a lot of those. 
But will add Google Cloud Catalog, that might be worth it.